pathd/path_pcep.c

```diff
@@ -248,31 +248,43 @@ DEFUN(show_pcep_counters, show_pcep_counters_cmd, "show pcep counters",
 }
 
 DEFUN_NOSH(pcep_cli_pcc, pcep_cli_pcc_cmd,
-	   "pcc [ip A.B.C.D] [port (1024-65535)] [force_stateless]",
+	   "pcc [ip A.B.C.D | ipv6 X:X::X:X] [port (1024-65535)] [force_stateless]",
```

Comment:
The way of doing this in FRR is `ip <A.B.C.D|X:X::X:X>` see https://github.com/opensourcerouting/frr/blob/pathd-pcep/bgpd/bgp_vty.c#L5014